File: c#/web/WebApplication1/Entity/NewsAlarm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Entity
{
    /// <summary>
    /// 新消息提醒实体类
    /// </summary>
    public class NewsAlarm
    {
        /// <summary>
        /// 消息创建时间
        /// </summary>
        public DateTime createDate { set; get; }

        /// <summary>
        /// 消息摘要
        /// </summary>
        public string title { set; get; }

        /// <summary>
        /// 查看新消息的菜单页。
        /// </summary>
        public string menuId { set; get; }

        /// <summary>
        /// 新消息的接受者
        /// </summary>
        public string receive { set; get; }

        /// <summary>
        /// 消息的读取状态：0：未读，1：已读
        /// </summary>
        public int readState { set; get; }
    }
}

File: java/web/src/main/java/web/template/entity/NewsAlarm.java
package web.template.entity;
import java.util.Date;
/**
 * 最新消息提醒
 * @author admin
 */
public class NewsAlarm {
	/**
	 * 提醒日期
	 */
	public Date createDate;

	/**
	 * 提醒标题
	 */
    public String title;

    /**
     * 菜单id
     */
    public String menuId;

    /**
     * 接收者
     */
    public String receive;

    /**
     * 读取状态
     */
    public Integer readState;

	public Date getCreateDate() {
		return createDate;
	}

	public void setCreateDate(Date createDate) {
		this.createDate = createDate;
	}

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getMenuId() {
		return menuId;
	}

	public void setMenuId(String menuId) {
		this.menuId = menuId;
	}

	public String getReceive() {
		return receive;
	}

	public void setReceive(String receive) {
		this.receive = receive;
	}

	public Integer getReadState() {
		return readState;
	}

	public void setReadState(Integer readState) {
		this.readState = readState;
	}
}
